### ISSUE TITLE
Functionality error

### ISSUE THREAD

**Issue by ShaylynHM** [ShaylynHM/pretty-products#1]
@hellachella
@carolsinc

Please help! I don't understand what these messages mean.

<img width="809" alt="Screen Shot 2021-04-09 at 4 50 22 PM" src="https://user-images.githubusercontent.com/77545465/114239620-4b9fd280-9954-11eb-8478-9b54e9134338.png">

**Comment by carolsinc** [comment]
Your aria-label should be on the a tag, not the img
